README.md

```diff
@@ -30,21 +30,28 @@ Here are the available commands in this repository:
 
 The site requires Node.js. You can use [`nvm`](https://github.com/nvm-sh/nvm) to automatically use the version of Node.js which is specified in the repository's [`.nvmrc`](./.nvmrc) file:
 
-```
+```sh
 nvm install
 ```
 
 ## How to run the site locally
 
 Install dependencies and run the development server:
 
-```
+```sh
 npm install
 npm run dev
 ```
-
 Open [http://localhost:3000](http://localhost:3000) with your browser to load the result.
 
+## Configure the `pre-commit` Githook
+
+To enable our pre-commit hook, update the following git config setting:
+
+```sh
+$ git config core.hooksPath .githooks
```

Comment:
Do you mind if we remove the `$` here? I think people will copy/paste

Reply:
100%!